scripts/deploy-dex.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from "hardhat";
+import { Contract, ContractFactory } from "ethers";
+import prompts from "prompts";
+
+async function main() {
+    const addrs = await ethers.getSigners();
+
+    console.log("Deploying contracts with the account:", addrs[0].address);
+    console.log("Account balance:", (await addrs[0].getBalance()).toString());
+
+
+    const { dex, name } = await prompts([{
```

Comment:
`name` param is not used?

Reply:
Maybe this was your idea and that's why you ask for the name, but I'd like the addition of the Dex contract to the UL to be part of this script, so after deployment it gets set up on the UL.